### PR TITLE
add link to cms instructions in admin header Closes #56

### PR DESCRIPTION
**Pull request by scottalexandra**
added CMS Instuctions link on the admin navbar
added a test to admin_dashboard_spec
All tests passing

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
+    get "/dashboard/cms-instructions", to: "dashboard#cms_instructions", as: "cms_instructions"
     resources :companies, except: [:create] do
       resources :notes, only: [:index, :create, :destroy]
     end
```
Comment (Lydias303) [inline config/routes.rb:22]: Do we want this to be its own page?

**Comment by scottalexandra** [comment]
Added cms_instructions action to admin dashboard controller
added cms_instructions view in admin/dashboard folder
added _cms_instructions partial
styled cms instructions page 
all tests passing!

File: config/routes.rb [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
+    get "/dashboard/cms-instructions", to: "dashboard#cms_instructions", as: "cms_instructions"
```
Comment (oorion) [inline config/routes.rb:15]: Looks great! :sparkle:

File: spec/features/admin_dashboard_spec.rb [hunk 3]
```diff
@@ -14,6 +14,17 @@ def login
       click_link_or_button('Log in')
     end
 
+    it "can click a link to view cms instructions" do
```
Comment (oorion) [inline spec/features/admin_dashboard_spec.rb:17]: Nice test!

File: spec/features/admin_dashboard_spec.rb [hunk 4]
```diff
@@ -14,6 +14,17 @@ def login
...
+      login
+      within('.navbar-header') do
+        click_link_or_button('CMS Instructions')
+      end
+      expect(current_path).to eq(admin_cms_instructions_path)
+      within('.cms-instructions-header') do
+        expect(page).to have_content('Content Management System(CMS) Instructions')
+      end
+    end
```
Comment (bhargavisatpathy) [inline spec/features/admin_dashboard_spec.rb:26]: Looks good to me too :+1:

**Pull request merged**